target_snowflake/file_formats/csv.py

```diff
@@ -45,30 +45,50 @@ def create_merge_sql(table_name: str,
            f"VALUES ({p_insert_values})"
 
 
+def prep_csv_row_string(flatten_record, column):
```

Comment:
I think this PR is open against the wrong branch. This code is available in both master and release.

Reply:
New branch has the latest